src/docs/asciidoc/http-client.adoc

```diff
@@ -429,6 +429,40 @@ public class Application {
 ----
 ====
 
+
+== Bootstrap Configuration 
+
+Use `HttpClient.from` and create a `TcpClient` when you need more control of your client configuration, i.e., when you 
+want to specify the local address.
+
+
+====
+[source,java]
+----
+import reactor.netty.http.client.HttpClient;
+import reactor.netty.tcp.TcpClient;
+import java.net.InetSocketAddress;
+
+public class Application {
+
+    public static void main(String[] args) {
+	    TcpClient tcpClient = TcpClient.create()
+					.bootstrap(b -> b.localAddress(new InetSocketAddress("host", port)));
```

Comment:
@violetagg isn't the `bootstrap` method now deprecated ?

Reply:
@simonbasle I wanted to do this change independent from this PR, but if you think it is better to do it here we can ask @lfmunoz to change it

Reply:
this PR was created before our deprecation of `bootstrap`

Reply:
ah got it. might as well do it in one go and directly introduce the most up to date snippet